src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -24,6 +24,7 @@
         "twig/twig": "~1.28|~2.0"
     },
     "require-dev": {
+        "symfony/console": "~3.3",
```

Comment:
We can also duplicate the code of this helper in the component and in the bundle, but I'm not sure it is worth it.